### DISCUSSION TITLE
Double click on a link or activate API

### DISCUSSION THREAD

**Discussion by zuan111** [AutomaApp/automa#1481] [Q&A]
HI,  I have two options on a chrome site we are using to monitor cameras which i need to refresh every few minutes. the camera feed runs for 180 seconds then needs to be refreshed on an API within the website.  

1.  I need to right click and select an option on an API within a website to refresh the API. THis is for a camera feed which i would like to refresh every 5 minutes. 
2.  I can double click on a link on the site to refresh the API. 

Can i have assistance how to make this happen with Automa? Cant find a trigger for this

**Comment by Delusionnnn** [comment]
You can send an API request with HTTP request block. I'm not sure if it can refresh but it should be able to. 

As for double click, use Trigger Event block.

![image](https://github.com/AutomaApp/automa/assets/141815049/b85aceef-0edf-491f-a913-ef13a6088fcb)
